packages/amplify-codegen/tests/commands/models.test.js

```diff
@@ -8,23 +8,24 @@ jest.mock('@graphql-codegen/core');
 const MOCK_CONTEXT = {
```

Comment:
The changes here are format-only conducted automatically by `prettier`. Please omit this for the review.